dev/io.openliberty.checkpoint_fat/publish/servers/webCacheServer/bootstrap.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+websphere.java.security.exempt=true
+com.ibm.ws.logging.trace.specification=*=info:checkpoint=all
+io.openliberty.checkpoint.dump.threads=true
+io.openliberty.checkpoint.allowed.features=webCache-1.0
```

Comment:
Must enable webCache-1.0 by adding the header `WLP-InstantOn-Enabled: true` to https://github.com/OpenLiberty/open-liberty/blob/fcb5a7c86f4329639936c3bda3cc18fd58fa8b51/dev/com.ibm.websphere.appserver.features/visibility/public/webCache-1.0/com.ibm.websphere.appserver.webCache-1.0.feature